e2e/e2eutil/utils.go

```diff
@@ -133,6 +133,26 @@ func WaitForAllocsRunning(t *testing.T, nomadClient *api.Client, allocIDs []stri
 	}
 }
 
+func WaitForAllocsNotPending(t *testing.T, nomadClient *api.Client, allocIDs []string) {
+	for _, allocID := range allocIDs {
+		WaitForAllocNotPending(t, nomadClient, allocID)
+	}
+}
+
+func WaitForAllocNotPending(t *testing.T, nomadClient *api.Client, allocID string) {
+	testutil.WaitForResultRetries(retries, func() (bool, error) {
+		time.Sleep(time.Millisecond * 100)
+		alloc, _, err := nomadClient.Allocations().Info(allocID, nil)
+		if err != nil {
+			return false, err
+		}
+
+		return alloc.ClientStatus != structs.AllocClientStatusPending, fmt.Errorf("expected status running, but was: %s", alloc.ClientStatus)
```

Comment:
The error message seems possibly misleading here: we're waiting for not-pending, not "running".

Reply:
woops thanks